Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyComboBox.cxx

```diff
@@ -64,6 +65,7 @@ class qMRMLSubjectHierarchyComboBoxPrivate
 qMRMLSubjectHierarchyComboBoxPrivate::qMRMLSubjectHierarchyComboBoxPrivate(qMRMLSubjectHierarchyComboBox& object)
   : q_ptr(&object)
   , MaximumNumberOfShownItems(20)
+  , AlignPopupVertically(true)
```

Comment:
Does this mean that the default is now changed?

Reply:
No, the default would align it vertically (selected item overlayed on top of combobox)

Reply:
OK, let's see which one seems better in the long run. Maybe the way you implemented it is more desirable to the users. I don't really have a preference right now, just wanted to know.